git-tar/function/ops.go

```diff
@@ -44,18 +45,28 @@ func parseYAML(pushEvent sdk.PushEvent, filePath string) (*stack.Services, error
 }
 
 func fetchTemplates(filePath string) error {
-	templateRepos := formatTemplateRepos()
-
-	for _, repo := range templateRepos {
-		pullCmd := exec.Command("faas-cli", "template", "pull", repo)
-		pullCmd.Dir = filePath
-		err := pullCmd.Start()
-		if err != nil {
-			return fmt.Errorf("Failed to start faas-cli template pull: %t", err)
+	if _, err := os.Stat(filePath + "/template"); os.IsNotExist(err) {
```

Comment:
I had to do a review on my own code, since I forgot how it exactly worked, I guess now with the custom templates feature, things get more complicated as I need to know if the template is already downloaded, what if we want to overwrite/update the existing templates, should there be environmental variable disabling that ?